docs/README.md

```diff
@@ -2,89 +2,140 @@
 title: XOD Documentation
 ---
 
+<style>
+/* Force linked headers to be black, not blue */
```

Comment:
If it's a general rule(s) — maybe move it into `styles.css`?

Reply:
No, it’s not a general rule. It can break some complex layouts done with Semantic UI. I mean, it could be in `style.css`, but a thorough research is required. I would like to postpone it until the moment we’d like to make some other headers black again.

Reply:
👌